src/sentry/static/sentry/app/views/releases/list/releaseHealth/content.tsx

```diff
@@ -72,8 +73,8 @@ const Content = ({
               </DailyColumn>
             </React.Fragment>
           )}
-          <CrashesColumn>{t('Crashes')}</CrashesColumn>
-          <IssuesColumn>{t('New Issues')}</IssuesColumn>
+          <RightColumn>{t('Crashes')}</RightColumn>
```

Comment:
I don't think we should call it "RightColumn", as the columns are not visually separated